docs/rules/no-deprecated-slot-attribute.md

```diff
@@ -43,16 +43,18 @@ This rule reports deprecated `slot` attribute in Vue.js v2.6.0+.
 ```json
 {
   "vue/no-deprecated-slot-attribute": ["error", {
-    "ignore": ["my-component"]
+    "ignore": ["my-component"],
+    "ignoreParents": ["my-web-component"],
   }]
 }
 ```
 
 - `"ignore"` (`string[]`) An array of tags or regular expression patterns (e.g. `/^custom-/`) that ignore these rules. This option will check both kebab-case and PascalCase versions of the given tag names. Default is empty.
+- `"ignoreParents"` (`string[]`) An array of tags or regular expression patterns (e.g. `/^custom-/`) for parents that ignore these rules. This option is especially useful for Web-Components. This option will check both kebab-case and PascalCase versions of the given tag names. Default is empty.
```

Comment:
I think web components can only be kebab-case, so should we really automatically check PascalCase-ified versions, too?

Reply:
This might be relevant for other things other than web components. Web components is just my usecase.

To be honest I wouldnt have added either kebab or pascal case (for ignore and ignoreParents) but I wanted to keep it consistent with the existing ignore option.

Reply:
It might, but I think this is pretty theoretical. This rule is reporting a _deprecated_ attribute after all. The already existing `ignore` option is there for people using `slot` as a prop or attribute name in certain components, so there catching both kebab-case and PascalCase notations with a single option makes sense. But I can't think of a reason (apart from web components) where you might ignore the `slot` attribute if a _parent_ (Vue) component has a certain name.

Reply:
So should we use only the given patterns and ignore/remove the explicit case conversions?
If web-components are the only expected value, then the user should only define patterns matching the web-component naming convention.

Reply:
Yes, I think that would be better.

Reply:
Done